icu4c/source/data/mappings/gb18030.ucm

```diff
@@ -28077,6 +28077,28 @@ CHARMAP
 <U9FA3> \xFD\x99 |0
 <U9FA4> \xFD\x9A |0
 <U9FA5> \xFD\x9B |0
+<U9FA6> \x82\x35\x8F\x33 |0
+<U9FA7> \x82\x35\x8F\x34 |0
+<U9FA8> \x82\x35\x8F\x35 |0
+<U9FA9> \x82\x35\x8F\x36 |0
+<U9FAA> \x82\x35\x8F\x37 |0
+<U9FAB> \x82\x35\x8F\x38 |0
+<U9FAC> \x82\x35\x8F\x39 |0
+<U9FAD> \x82\x35\x90\x30 |0
+<U9FAE> \x82\x35\x90\x31 |0
+<U9FAF> \x82\x35\x90\x32 |0
+<U9FB0> \x82\x35\x90\x33 |0
+<U9FB1> \x82\x35\x90\x34 |0
+<U9FB2> \x82\x35\x90\x35 |0
+<U9FB3> \x82\x35\x90\x36 |0
+<U9FB4> \xFE\x59 |0
```

Comment:
Please do not add mappings that are covered by the algorithmic ranges. This change should only modify mappings that are outside the ranges or whose mappings differ from what the code would do based on the algorithmic ranges.

Reply:
Yes, as noted, I misunderstood the order of handling for ranges vs explicit